specification/securityinsights/resource-manager/common/v1/types.json

```diff
@@ -0,0 +1,113 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "CloudError": {
```

Comment:
>CloudError [](start = 5, length = 10)

Similar definitions already exist in v2 common types. Please look into switching over to that version.

Reply:
Not totally the same, don't want to switch right now